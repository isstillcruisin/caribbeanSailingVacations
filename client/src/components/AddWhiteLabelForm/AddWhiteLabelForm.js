import React from 'react'

const AddWhiteLabelForm = props => (
  <form>
    <div className="form-group add-white-label-form">
      <label htmlFor="white-label-name">
        <strong>White-Label Name</strong>
      </label>
      <input
        className="form-control"
        id="white-label-name"
        type="text"
        value={props.whiteLabelName}
        placeholder="name of the white-label website..."
        name="whiteLabelName"
        onChange={props.handleInputChange}
        required
      />
    </div>
    <div className="pull-right">
      <button
        onClick={props.handleFormSubmit}
        type="submit"
        className="btn btn-lg btn-danger"
      >
        Submit
      </button>
    </div>
  </form>
)

export default AddWhiteLabelForm
