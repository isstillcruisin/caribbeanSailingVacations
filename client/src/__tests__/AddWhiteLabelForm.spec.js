import React from 'react'
import { create } from 'react-test-renderer'
import AddWhiteLabelForm from '../components/AddWhiteLabelForm'

describe('AddWhiteLabelForm component', () => {
  test('Matches the snapshot', () => {
    const form = create(<AddWhiteLabelForm whiteLabelName='test' />)
    expect(form.toJSON()).toMatchSnapshot()
  })
})