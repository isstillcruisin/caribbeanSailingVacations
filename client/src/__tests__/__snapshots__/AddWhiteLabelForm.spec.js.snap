// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddWhiteLabelForm component Matches the snapshot 1`] = `
<form>
  <div
    className="form-group add-white-label-form"
  >
    <label
      htmlFor="white-label-name"
    >
      <strong>
        White-Label Name
      </strong>
    </label>
    <input
      className="form-control"
      id="white-label-name"
      name="whiteLabelName"
      placeholder="name of the white-label website..."
      required={true}
      type="text"
      value="test"
    />
  </div>
  <div
    className="pull-right"
  >
    <button
      className="btn btn-lg btn-danger"
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
